--- glib-2.88.0/meson.build.orig	2026-04-12 14:08:20.636817232 +0200
+++ glib-2.88.0/meson.build	2026-04-12 14:09:45.299004040 +0200
@@ -720,8 +720,8 @@ functions = [
   'fchmod',
   'fchown',
   'fdwalk',
-  'free_aligned_sized',
-  'free_sized',
+  #  'free_aligned_sized',
+  #'free_sized',
   'fsync',
   'ftruncate64',
   'getauxval',
@@ -847,7 +847,7 @@ endforeach
 
 # Export the information about free_sized() so we can correctly define a macro
 # wrapper around g_free()/g_free_sized() depending on whether it’s available
-glibconfig_conf.set('G_HAVE_FREE_SIZED', have_func_free_sized)
+#glibconfig_conf.set('G_HAVE_FREE_SIZED', have_func_free_sized)
 
 # Check that stpcpy() is usable; must use header.
 # See:
