Index: package/python/python-installer/python-installer.desc
===================================================================
--- package/python/python-installer/python-installer.desc	(revision 79711)
+++ package/python/python-installer/python-installer.desc	(working copy)
@@ -21,7 +21,7 @@
 [L] MIT
 [S] Stable
 [V] 0.7.0
-[P] X -----5---9 109.001
+[P] X 0----5---9 109.001
 
 [D] 36f2f1a09c0e75452b58d01f7d9d105f24713ce04d311a609214fdf0 python-installer-0.7.0.zip !https://files.pythonhosted.org/packages/py3/i/installer/installer-0.7.0-py3-none-any.whl
 
@@ -34,7 +34,7 @@
 hook_add inmake 5 pyinstall_inmake
 
 pyinstall_inmake() {
-	local sitedir=$root$libdir/python/site-packages
+	local sitedir=$root$libdir/python3.13/site-packages
 	mkdir -p $sitedir
 	cp -a installer* $sitedir
 	python3 -m compileall $sitedir
